imports/plugins/core/ui/client/components/button/buttonSelect.js

```diff
@@ -0,0 +1,133 @@
+import React, { Component, PropTypes } from "react";
+import classnames from "classnames";
+import Button from "./button.jsx";
+
+class ButtonSelect extends Component {
+  static PropTypes = {
+    buttons: PropTypes.array,
+    currentButton: PropTypes.node,
+    defaultButton: PropTypes.object,
+    defaultNonActiveButtons: PropTypes.array,
+    nonActiveButtons: PropTypes.array
+  }
+
+  state = {
+    toggle: "hidden",
+    currentButton: {},
+    buttons: [],
+    activeButton: "",
+    nonActiveButtons: [],
+    defaultBgClassNames: "",
+    toggleIcon: classnames({ "fa": true, "fa-chevron-down": true, "text-center": true, "fa-icon": true }),
+    toggleClassNames: classnames({ "button-dropdown": true, "hidden": true })
+  }
+
+  componentWillMount() {
+    this.handleDefaultState();
+  }
+
+  handleDefaultState = () => {
+    const props = this.props;
+    let defaultButton = props.buttons.filter(button => {
+      if (button.active === true) {
+        return button;
+      }
+    });
+    defaultButton = defaultButton[0];
+
+    const defaultBgClassNames = classnames({ "button-select": true, [defaultButton.bgColor]: true });
+
+    const defaultNonActiveButtons = props.buttons.filter(button => {
+      if (button.active === false || button.active === undefined) {
+        return button;
+      }
+    });
+    const currentButton = <Button
```

Comment:
Indentation is a but weird, try something like following. (kinda like how you did the return in the render)
```
const currentButton = (
  <Button
    yourPropsAndStuff
  >
  </Button>
);
```